staging_service/import_specifications/individual_parsers.py

```diff
@@ -352,8 +357,14 @@ def parse_dts_manifest(path: Path) -> ParseResults:
         with open(path, "r") as manifest:
             manifest_json = json.load(manifest)
         if not isinstance(manifest_json, dict):
-            errors.append(Error(ErrorType.PARSE_FAIL, "Manifest is not a dictionary", spcsrc))
-
+            return _error(Error(ErrorType.PARSE_FAIL, "Manifest is not a dictionary", spcsrc))
+        validator = jsonschema.Draft202012Validator(dts_manifest_schema)
```

Comment:
Can this line also be moved to the config section, so this function expects a validator class? Then the warning about invalid schemas should be moot

Reply:
Done.

Reply:
:+1: